python/core/auto_generated/fieldformatter/qgsvaluerelationfieldformatter.sip.in

```diff
@@ -59,7 +59,7 @@ Constructor for QgsValueRelationFieldFormatter.
 
     static QStringList valueToStringList( const QVariant &value );
 %Docstring
-Utility to convert an array or a string representation of an array ``value`` to a string list
+Utility to convert an array or a string representation of an (C style: {1,2...}) array in the form ``value`` to a string list
```

Comment:
Should that be hstore instead of C style? (and list instead of array since it's either `QVariantList` or python `list`)?

No big issue, just wondering.